stateresolutionv2.go

```diff
@@ -46,15 +46,20 @@ type stateResolverV2 struct {
 	resolvedMembers           map[spec.SenderID]PDU         // Resolved member events
 	resolvedOthers            map[StateKeyTuple]PDU         // Resolved other events
 	result                    []PDU                         // Final list of resolved events
+	isRejected                IsRejected                    // Check if the given eventID is rejected
 }
 
+// IsRejected should return if the given eventID is rejected or not.
+type IsRejected func(eventID string) bool
+
 // ResolveStateConflicts takes a list of state events with conflicting state
 // keys and works out which event should be used for each state event. This
 // function returns the resolved state, including unconflicted state events.
 func ResolveStateConflictsV2(
-	conflicted, unconflicted []PDU,
+	conflicted, unconflicted,
 	authEvents []PDU,
 	userIDForSender spec.UserIDForSender,
+	isRejected IsRejected,
```

Comment:
Please rename this to `isRejectedFn` everywhere. I got very confused and though this was a `bool` which doesn't make sense.